packages/eslint-plugin/lib/rules/boolean-literal-compare.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @fileoverview Flags unnecessary equality comparisons against booleans
```

Comment:
Sorry we've changed to using all-contributors now and removed all other occurrences of these file-level author comments, please could you remove this